docs/scss/components/_example.scss

```diff
@@ -75,6 +75,16 @@ rh-alert + .example {
 //   margin-bottom: 0;
 // }
 
+.example--palette-none {
+  padding: 0;
+  border: none;
+  border-radius: none;
+
```

Comment:
In the navigation secondary docs the images that would use this look and feel are not centered.  I wonder if this is true all the time or if there should be an additional attribute for inline-start alignment.

```Suggestion

  img:only-child {
    margin: 0;
  }
  
  ```